web/src/user_profile.js

```diff
@@ -269,6 +269,15 @@ function render_manage_profile_content(user) {
     }
 }
 
+function get_tippy_content(field_name) {
+    const hyperlink = /\[([^[\]]*?)]\((.*?)\)/g;
+    const mentions = /\*{1,2}([^*]+)\*{1,2}/g;
```

Comment:
Does this cover all the various markdown content which a field name can have?

Reply:
it now covers mentions, silent mentions, bold/italic, url, <time> fields. emojis are anyways shown as `:smile:` so I guess yeah?